From cd609008b272472678b47a5f4ac54568430a4239 Mon Sep 17 00:00:00 2001
From: Manoj Srivastava <srivasta@debian.org>
Date: Fri, 30 Oct 2009 15:10:20 -0500
Subject: [PATCH 4/4] Fix an error due to misintertpreting .MT/*

Signed-off-by: Manoj Srivastava <srivasta@debian.org>
---
 mcon/man/mconfig.SH | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mcon/man/mconfig.SH b/mcon/man/mconfig.SH
index 7013904..91bafee 100755
--- a/mcon/man/mconfig.SH
+++ b/mcon/man/mconfig.SH
@@ -443,7 +443,7 @@ he is going to use the final product on.
 Metaconfig reserves the following names in your directory, and if you use such
 a name it may get clobbered or have other unforeseen effects:
 .Ex
-.MT/*
+ \.MT/*
 Configure
 Wanted
 Obsolete
-- 
2.0.0.rc0

